chat/src/test/java/SpaceManageMessageTestIT.java

```diff
@@ -0,0 +1,172 @@
+    $(byText(usernamea + " " + usernamea)).parent().find(ELEMENT_ICON_ACCEPT_SPACE_REQUEST_IN_MEMBERS_TAB).click();
+    homePagePlatform.goToHomePage();
+    homePagePlatform.goToChat();
+    switchTo().window("Chat");
+    info("send message on space");
+    chatManagement.sendMessageInRoomOrSpace(space, message);
+    switchTo().window("Home Page");
+    manageLogInOut.signIn(usernamea, password);
+    homePagePlatform.goToChat();
+    switchTo().window("Chat");
+    $(byText(space)).click();
+    info("verify message");
+    ELEMENT_CHAT_LIST_MSG.find(byText(message)).should(Condition.exist);
+    switchTo().window("Home Page");
+    manageLogInOut.signIn("root", "gtn");
+    info("delete data");
+    homePagePlatform.goToAllSpace();
+    spaceManagement.deleteSpace(space, false);
+    navigationToolbar.goToManageCommunity();
+    userandgroupmanagement.deleteUser(usernamea);
+  }
+
+  @Test
+  public void test02_checkSpaceChatDeletedAfterDelete() {
+    String space = "space" + getRandomNumber();
+    homePagePlatform.goToMySpaces();
+    spaceManagement.addNewSpaceSimple(space, space);
+    homePagePlatform.goToHomePage();
+    homePagePlatform.goToChat();
+    switchTo().window("Chat");
+    info("check that space exist");
+    $(byText(space)).should(Condition.exist);
+    switchTo().window("Home Page");
+    homePagePlatform.goToAllSpace();
+    spaceManagement.deleteSpace(space, false);
+    switchTo().window("Chat");
+    refresh();
+    info("check that space chat deleted");
+    $(byText(space)).shouldNot(Condition.exist);
+  }
+
+  @Test
+  public void test03_sendMessageAndItsAnswerFromOtherUser() {
+    String usernamea = "usernamea" + getRandomString();
+    String password = "123456";
+    String emaila = usernamea + getRandomNumber() + "@test.com";
+    String message = "message" + getRandomNumber();
+    String message2 = "messagee" + getRandomNumber();
+    navigationToolbar.goToAddUser();
+    info("Create new user");
+    userAddManagement.addUser(usernamea, password, emaila, usernamea, usernamea);
+    manageLogInOut.signIn(usernamea, password);
+    homePagePlatform.goToChat();
+    switchTo().window("Home Page");
+    manageLogInOut.signIn("root", "gtn");
+    homePagePlatform.goToHomePage();
+    switchTo().window("Chat");
+    refresh();
+    ELEMENT_CHAT_INPUT_SEARCH_USER.setValue("@" + usernamea);
+    $(byText(usernamea + " " + usernamea)).click();
+    info("root send message to user");
+    chatManagement.sendMessageInRoomOrSpace(usernamea + " " + usernamea, message);
+    switchTo().window("Home Page");
+    manageLogInOut.signIn(usernamea, password);
+    homePagePlatform.goToChat();
+    switchTo().window(1);
+    $(byText("Root Root")).click();
+    $(byText(message)).should(Condition.exist);
+    info("user check message and answer root");
+    chatManagement.sendMessageInRoomOrSpace("Root Root", message2);
+    switchTo().window("Home Page");
+    manageLogInOut.signIn("root", "gtn");

```

Comment:
don't use strings to sign in, if the default root user is updated, this test will fail.